keg_auth/model/__init__.py

```diff
@@ -357,12 +376,17 @@ def username(cls):
 
     @hybrid_property
     def is_active(self):
-        return self.is_verified and self.is_enabled
+        is_disabled = self.disable_date is not None and self.disable_date < arrow.utcnow()
```

Comment:
`UserEmailMixin` is assumed to be used alongside `UserMixin`. So, when we create the additional hybrid property above, we can utilize it here and in the expression version below.